tmt-app/src/Components/ResultsCards.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Stack } from "@mui/material";
+import CustomCard from "./CustomCard";
+
+function ResultsCards() {
+  const dummyData = [
+    {
+      id: 0,
+      name: "Education",
+      desc: "testing",
+      icon: "emotional_intelligence",
+    },
+    {
+      id: 1,
+      name: "Economics and Business",
+      desc: "test fdasf afas fsadf dsf dsf edsf ds fsdaf sd f asfsd fea fgdsffdsf dfasdf ds fsdafsdf fgsdaf ased fasd few gasef sWEAFGASDFAS FDFSA DFA FSADF EAFSDASD FGREAASDFG EF GRasdfga gadfaefadf asf ",
+      icon: "chemistry",
+    },
+  ];
+  return (
+    <Stack
+      spacing={3}
+      sx={{
+        width: "100%",
+        "&::after": {
+          content: "''",
+          width: "90%",
+          margin: "3rem auto",
+          border: "1px solid black",
+        },
```

Comment:
Can you explain what this style is for?

Reply:
This is for the solid line after the section. 
![image](https://user-images.githubusercontent.com/36683390/231365780-d74e9d1c-955d-4cda-a074-e990c432c253.png)

Reply:
Let's use a new element like the horizontal rule to also allow the context of this section break to exist in the code.

When looking at a page, we have different conceptual "layers", display + context in this case. We're telling the user this is the end of the section, but using styles to do so. But users using screen readers would never be able to tell. So in the context layer we could use a horizontal rule to indicate the ending of a section that a screen reader would see and communicate to the user